src/chunk.h

```diff
@@ -25,6 +25,17 @@
 #define DROP_CHUNKS_NARGS 4
 #define COMPRESS_CHUNK_FUNCNAME "compress_chunk"
 #define COMPRESS_CHUNK_NARGS 2
+#define DECOMPRESS_CHUNK_FUNCNAME "decompress_chunk"
+#define RECOMPRESS_CHUNK_FUNCNAME "recompress_chunk"
+#define RECOMPRESS_CHUNK_NARGS 2
+
+typedef enum ChunkCompressionStatus
+{
+	CHUNK_COMPRESS_NONE = 0,
```

Comment:
So we have this enum and then we have `CHUNK_STATUS_UNORDERED` type #defines as well..

Reply:
The `#define`s are part of the database protocol and ideally will never change. They describe the chunk status column. This is an internal in-memory state for some limited scope logic here. It's mostly to separate the cases of the return values of `ts_chunk_get_compression_status()`.

My suggestion was to change `CHUNK_DROPPED` to `CHUNK_COMPRESS_DROPPED` for this reason.